ledger/ledger.go

```diff
@@ -94,7 +94,8 @@ func GetLedgerComponents(conf configuration.Ledger) []interface{} {
 		artifactmanager.NewArtifactManger(db),
 		jetcoordinator.NewJetCoordinator(db, conf.JetCoordinator),
 		pulsemanager.NewPulseManager(db, conf.PulseManager),
-		artifactmanager.NewMessageHandler(db, storage.NewRecentStorage(1), &conf.ArtifactManager),
+		storage.NewRecentStorage(1),
```

Comment:
Let's move 1 to config. And it probably should be around 50

Reply:
Agree